pom.xml

```diff
@@ -56,8 +56,19 @@
                 <groupId>org.ballerinalang</groupId>
                 <artifactId>ballerina-runtime</artifactId>
                 <version>${ballerina.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.apache.james</groupId>
+                        <artifactId>apache-mime4j-core</artifactId>
+                    </exclusion>
+                </exclusions>
                 <scope>compile</scope>
             </dependency>
+            <dependency>
+                <groupId>org.apache.james</groupId>
+                <artifactId>apache-mime4j-core</artifactId>
+                <version>0.8.9</version>
```

Comment:
Shall we extract the mime4j version into a configuration property ?

Reply:
Done